packages/client/hmi-client/src/page/project/components/tera-resource-sidebar.vue

```diff
@@ -137,6 +143,7 @@ import AccordionTab from 'primevue/accordiontab';
 import Button from 'primevue/button';
 import { IProject, ProjectAssetTypes, ProjectPages, isProjectAssetTypes } from '@/types/Project';
 import { useDragEvent } from '@/services/drag-drop';
+import 'primeicons/primeicons.css';
```

Comment:
This is imported at the top level of the application right? 

Why do we need a separate import here?

Reply:
You're right.. it seems to work without it